cmd/oceantv/broadcast_manager_test.go

```diff
@@ -0,0 +1,114 @@
+package main
+
+import (
+	"testing"
+	"time"
+
+	"github.com/ausocean/cloud/cmd/oceantv/broadcast"
+)
+
+func TestBroadcastCanBeReused(t *testing.T) {
```

Comment:
Might you also be able to add an integration test using the broadcastSystem ?

Reply:
Here's an example

https://github.com/ausocean/cloud/blob/main/cmd/oceantv/broadcast_machine_test.go#L1348

Reply:
I did have a look at writing an integration test but for CreateBroadcast, it needs a token bucket limiter in the store or something and I wasn't sure how to imitate that.
I get this: could not get token bucket limiter: could not unmarshal token bucket limiter: unexpected end of JSON input

Reply:
David has mentioned that he did something with options on this PR: https://github.com/ausocean/cloud/pull/267/files

I'll have a look and see what I can do with options because I'm not quite familiar with how I can modify the test system with options.

Reply:
I think I'll leave this for now since it's somewhat of an urgent fix and it's taking some time for me to figure out the system testing stuff. That can happen in another PR if need be.

Reply:
As discussed I think we can deploy fixes before they are merged. I think this change deserves some integration testing to go along with it.

Reply:
See TestCreateBroadcast, it uses an actual broadcast manager so it runs the CreateBroadcast code.